index.js

```diff
@@ -238,3 +238,20 @@ const workdetailsSection = (content) => `
 }) 
 });
 }
+
+
```

Comment:
- [ ] great job. you are missing the javascript linter check(Eslint). ESLint helps you find and fix problems with your JavaScript code. you can see how to add that [here](https://github.com/microverseinc/linters-config/tree/master/javascript)